demo-project/conf/base/catalog_09_tracking.yml

```diff
@@ -1,9 +1,9 @@
-train_evaluation.linear_regression:
+train_evaluation.linear_regression.r2_score:
```

Comment:
Why does this need to be changed? Was this a problem before?

Reply:
it was wrong before and I am not sure since how long. on the pipeline the dataset is called r2_score but because it's not correctly defined in the catalog. these were getting stored as MemoryDataSets.